app.js

```diff
@@ -72,6 +82,18 @@ app.get('/', function(req, res) {
   });
 });
 
+app.get('/api/monitor', function(req, res) {
+  mjs.get('alivejson', function(err, v) {
+    if (v) {
+      let data = JSON.parse(v);
+      Object.entries(data).map(arr => { data[arr[0]] = Number(arr[1]); });
```

Comment:
I think it makes more sense to store the data as numbers than to cast it here. JSON supports numbers, e.g.:
```
JSON.parse(JSON.stringify({ a: 1, b: 2 }))
```
Tangent: I think your intention here would be clearer with `forEach` since you're don't end up using the return value of `map`.

Reply:
Fixed in 6aa0317 by just storing as Number before caching.